config/default/kustomization.yaml

```diff
@@ -1,39 +1,27 @@
 # Adds namespace to all resources.
-namespace: mig-controller-system
+namespace: mig
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: mig-controller-
+# namePrefix: mig
```

Comment:
Default kustomize setup for kubebuilder wants to prefix the namespace. I don't know if we've settled on the namespace we're going to put everything in, but using just plain `mig` for now.